aws-codeartifact-domain/src/main/java/software/amazon/codeartifact/domain/Translator.java

```diff
@@ -182,8 +182,16 @@ static ResourceModel translateFromReadResponse(
    * @return awsRequest the aws service request to delete a resource
    */
   static DeleteDomainRequest translateToDeleteRequest(final ResourceModel model) {
```

Comment:
can you add a unit test similar to the repo one here?

https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-codeartifact/blob/main/aws-codeartifact-repository/src/test/java/software/amazon/codeartifact/repository/ReadHandlerTest.java#L187

Also a comment would be nice explaining that this case happens because resource needs to be able to be deleted using only the identifier, in this case, the domainArn

Reply:
Thank you for pointing out, I've added a unit tests in PR #68